src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ConditionsCallout.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../../../../../Models/Services/StoryUtilities';
+import ConditionsCallout from './ConditionsCallout';
+import { IConditionsCalloutProps } from './ConditionsCallout.types';
+
+const wrapperStyle = { width: '100%', height: '600px', padding: 8 };
+
```

Comment:
is there really only one scenario here to validate? I can't tell since the story won't load, but just asking to prompt the thinking.